Mathlib/Analysis/Normed/Module/Dual.lean

```diff
@@ -306,3 +308,32 @@ theorem sInter_polar_eq_closedBall {𝕜 E : Type*} [RCLike 𝕜] [NormedAddComm
 end PolarSets
 
 end NormedSpace
+
+namespace LinearMap
+
+section NormedField
+
+variable {𝕜 E F : Type*}
+variable [NormedField 𝕜] [NormedSpace ℝ 𝕜] [AddCommMonoid E] [AddCommMonoid F]
+variable [Module 𝕜 E] [Module 𝕜 F]
+
+variable {B : E →ₗ[𝕜] F →ₗ[𝕜] 𝕜} (s : Set E)
+
+variable [Module ℝ F] [IsScalarTower ℝ 𝕜 F] [IsScalarTower ℝ 𝕜 𝕜]
+
+theorem polar_AbsConvex : AbsConvex 𝕜 (B.polar s) := by
+  rw [polar_eq_biInter_preimage]
+  exact AbsConvex.iInter₂ fun i hi =>
+    ⟨Balanced.mulActionHom_preimage (E := F) (balanced_closedBall_zero (E := 𝕜) (r := (1 : ℝ)))
+      (𝕜 := 𝕜) (B i), Convex.linear_preimage (convex_closedBall _ _) (B i)⟩
+
+/-
+TODO: prove the converse and upgrade this to the bipolar theorem
+-/
+set_option linter.unusedSectionVars false in
```

Comment:
I don't think that's what you want to do. Are you sure you want these unused section variables as hypotheses?

Reply:
I've removed this - we can come back to it in #20843